includes/rest-api/class-sensei-rest-api-question-helpers-trait.php

```diff
@@ -629,7 +634,7 @@ private function get_boolean_schema(): array {
 					'randomOrder'    => [
 						'type'        => 'boolean',
 						'description' => 'Should options be randomized when displayed to quiz takers',
-						'default'     => false,
+						'default'     => true,
```

Comment:
This doesn't seem to be the case when creating a new question (and I don't think it was before, either?).

Reply:
The block had that default for the setting control (based on #3760) — just wasn't working for saving, as the block won't set that attribute until it's modified. I updated the API so that 'not set' means yes, looks like the frontend already worked like that:

https://github.com/Automattic/sensei/blob/0f62fc8c51b39893e817a2b85aa7a1b3d5f4f771/includes/class-sensei-lesson.php#L1074-L1077

Reply:
Added some more fixes around this, should work consistently now.

Reply:
This is a change from the legacy editor (default=checked). Do we want that?